cloudbuild-k8s.yaml

```diff
@@ -20,35 +25,77 @@ substitutions:
   _CLUSTER_LOCATION: us-central1
 steps:
 
-- id: Build C2D Docker Image
+- id: Pull Dev Image
   name: gcr.io/cloud-builders/docker
+  waitFor:
+  - '-'
+  dir: k8s
+  entrypoint: bash
   args:
-  - build
-  - --tag
-  - click-to-deploy
-  - --file
-  - k8s.Dockerfile
-  - .
-
-- id: Run Cloud Build K8s Generator
-  name: click-to-deploy
-  entrypoint: python
-  args:
-  - scripts/cloudbuild_k8s_generator.py
-  - --solution
-  - $_SOLUTION_NAME
-  - --output
-  - k8s/$_SOLUTION_NAME/cloudbuild.yaml
+  - -exc
+  - |
+    TAG="$$(cat ./MARKETPLACE_TOOLS_TAG)"
+    docker pull "gcr.io/cloud-marketplace-tools/k8s/dev:$$TAG"
+    docker tag "gcr.io/cloud-marketplace-tools/k8s/dev:$$TAG" "gcr.io/cloud-marketplace-tools/k8s/dev:local"
 
-- id: Test Solution
+- id: Initialize Credentials
   name: gcr.io/cloud-builders/gcloud
```

Comment:
I agree that this is not ideal. Instead of running a new Cloud Build build, we could run it locally - https://cloud.google.com/cloud-build/docs/build-debug-locally.

I don't remember the details now, but this was not supported (maybe now it is).

Reply:
It is simpler to use the infrastructure to run the builds, and ten concurrent builds should be fine for us